### DISCUSSION TITLE
What web framework is used to build the Zustand documentation?

### DISCUSSION THREAD

**Discussion by lightzane** [pmndrs/zustand#2415] [General]
Hey, I just found out about the Zustand documentation, looks similar as if it were built with docusaurus, but it doesn't, or is it docusaurus or nextra?. Neither of those packages exists on the `package.json`.

I just wanted to know what is the web framework, etc, that was used to build the documentation site, and that includes just writing the markdown files, right?

Also, what is the theme for the code syntax highlighter that was used?

**Comment by dai-shi** [comment]
It's open source: https://github.com/pmndrs/website/tree/docs

**Reply by lightzane** [reply]
Is it this the one being used as the style theme for the prism.js / code syntax highlighter?

https://github.com/pmndrs/website/blob/docs/src/pages/pmndrs.css

**Reply by lightzane** [reply]
I also found a vscode ext for the `poimandres` theme ♥

The **red** font just don't look well with winston logger color.

![image](https://github.com/pmndrs/zustand/assets/8446145/2cc7a40f-b7de-4076-a2a5-fc3558b19250)
